Error during Run-AlPipeline?

**Describe the issue**
In the Run-AlPipeline process, I see errors within PipeLine, but it doesn't seem to disrupt the pipeline. Not quite sure if this is really the problem.
BC24 latest US

**Scripts used to create container and cause the issue**
```
Run-AlPipeline

ArtifactUrl                     https://bcartifacts.azureedge.net/sandbox/24.0.16410.18040/us
BuildBy                         BcContainerHelper,6.0.12


2024-04-04T00:43:46.3808373Z ##[section]Starting: Initialize job
2024-04-04T00:43:46.3812486Z Agent name: 'Hosted Agent'
2024-04-04T00:43:46.3812785Z Agent machine name: 'fv-az131-104'
2024-04-04T00:43:46.3812940Z Current agent version: '3.236.1'
2024-04-04T00:43:46.3851471Z ##[group]Operating System
2024-04-04T00:43:46.3851646Z Microsoft Windows Server 2022
2024-04-04T00:43:46.3851737Z 10.0.20348
2024-04-04T00:43:46.3851809Z Datacenter
2024-04-04T00:43:46.3851907Z ##[endgroup]
2024-04-04T00:43:46.3851988Z ##[group]Runner Image
2024-04-04T00:43:46.3852083Z Image: windows-2022
2024-04-04T00:43:46.3852173Z Version: 20240322.1.0
2024-04-04T00:43:46.3852378Z Included Software: https://github.com/actions/runner-images/blob/win22/20240322.1/images/windows/Windows2022-Readme.md
2024-04-04T00:43:46.3852628Z Image Release: https://github.com/actions/runner-images/releases/tag/win22%2F20240322.1
2024-04-04T00:43:46.3852772Z ##[endgroup]
2024-04-04T00:43:46.3852883Z ##[group]Runner Image Provisioner
2024-04-04T00:43:46.3853148Z 2.0.361.1
2024-04-04T00:43:46.3853252Z ##[endgroup]
2024-04-04T00:43:46.3854285Z Current image version: '20240322.1.0'
2024-04-04T00:43:46.3863357Z Agent running as: 'VssAdministrator'
2024-04-04T00:43:46.3912549Z Prepare build directory.
2024-04-04T00:43:46.4285021Z Set build variables.
2024-04-04T00:43:46.4322442Z Download all required tasks.
2024-04-04T00:43:46.4531178Z Downloading task: PowerShell (2.237.5)
2024-04-04T00:43:47.7946444Z Downloading task: PublishTestResults (2.237.0)
2024-04-04T00:43:48.7569093Z Downloading task: PublishBuildArtifacts (1.231.1)
2024-04-04T00:43:49.4349189Z Checking job knob settings.
2024-04-04T00:43:49.4360757Z    Knob: DockerActionRetries = true Source: $(VSTSAGENT_DOCKER_ACTION_RETRIES) 
2024-04-04T00:43:49.4361615Z    Knob: AgentToolsDirectory = C:\hostedtoolcache\windows Source: ${AGENT_TOOLSDIRECTORY} 
2024-04-04T00:43:49.4362273Z    Knob: UseGitLongPaths = true Source: $(USE_GIT_LONG_PATHS) 
2024-04-04T00:43:49.4362821Z    Knob: AgentPerflog = c:\vsts\perflog Source: ${VSTS_AGENT_PERFLOG} 
2024-04-04T00:43:49.4363509Z    Knob: AgentEnablePipelineArtifactLargeChunkSize = true Source: $(AGENT_ENABLE_PIPELINEARTIFACT_LARGE_CHUNK_SIZE) 
2024-04-04T00:43:49.4364516Z    Knob: ContinueAfterCancelProcessTreeKillAttempt = true Source: $(VSTSAGENT_CONTINUE_AFTER_CANCEL_PROCESSTREEKILL_ATTEMPT) 
2024-04-04T00:43:49.4364892Z    Knob: ProcessHandlerSecureArguments = false Source: $(AZP_75787_ENABLE_NEW_LOGIC) 
2024-04-04T00:43:49.4365148Z    Knob: ProcessHandlerSecureArguments = false Source: $(AZP_75787_ENABLE_NEW_LOGIC_LOG) 
2024-04-04T00:43:49.4365376Z    Knob: ProcessHandlerTelemetry = true Source: $(AZP_75787_ENABLE_COLLECT) 
2024-04-04T00:43:49.4365621Z    Knob: ProcessHandlerEnableNewLogic = true Source: $(AZP_75787_ENABLE_NEW_PH_LOGIC) 
2024-04-04T00:43:49.4366067Z    Knob: IgnoreVSTSTaskLib = true Source: $(AZP_AGENT_IGNORE_VSTSTASKLIB) 
2024-04-04T00:43:49.4366287Z    Knob: FailJobWhenAgentDies = true Source: $(FAIL_JOB_WHEN_AGENT_DIES) 
2024-04-04T00:43:49.4366604Z    Knob: CheckForTaskDeprecation = true Source: $(AZP_AGENT_CHECK_FOR_TASK_DEPRECATION) 
2024-04-04T00:43:49.4366831Z    Knob: MountWorkspace = true Source: $(AZP_AGENT_MOUNT_WORKSPACE) 
2024-04-04T00:43:49.4367126Z    Knob: LogTaskNameInUserAgent = true Source: $(AZP_AGENT_LOG_TASKNAME_IN_USERAGENT) 
2024-04-04T00:43:49.4367253Z Finished checking job knob settings.
2024-04-04T00:43:49.4986744Z Start tracking orphan processes.
2024-04-04T00:43:49.5078386Z ##[section]Finishing: Initialize job

```

**Full output of scripts**
```

Ready for connections!
Installing SqlServer module
##[error]Install-Package: C:\Run\my\HelperFunctions.ps1:5
Line |
   5 |  … er module"; Install-package SqlServer -Force -RequiredVersion 21.1.18 …
     |                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     | No match was found for the specified search criteria and package name
     | 'SqlServer'. Try Get-PackageSource to see all available registered
     | package sources.
Reading CustomSettings.config from hostedagent-bluebearwaste-ci
Installing SqlServer module
##[error]Install-Package: C:\Run\my\HelperFunctions.ps1:5
Line |
   5 |  … er module"; Install-package SqlServer -Force -RequiredVersion 21.1.18 …
     |                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     | No match was found for the specified search criteria and package name
     | 'SqlServer'. Try Get-PackageSource to see all available registered
     | package sources.
Creating Desktop Shortcuts for hostedagent-bluebearwaste-ci
Installing SqlServer module
##[error]Install-Package: C:\Run\my\HelperFunctions.ps1:5
Line |
   5 |  … er module"; Install-package SqlServer -Force -RequiredVersion 21.1.18 …
     |                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     | No match was found for the specified search criteria and package name
     | 'SqlServer'. Try Get-PackageSource to see all available registered
     | package sources.
Cleanup old dotnet core assemblies
Container hostedagent-bluebearwaste-ci successfully created
Installing SqlServer module

Use:
Get-BcContainerEventLog -containerName hostedagent-bluebearwaste-ci to retrieve a snapshot of the event log from the container
Get-BcContainerDebugInfo -containerName hostedagent-bluebearwaste-ci to get debug information about the container
Enter-BcContainer -containerName hostedagent-bluebearwaste-ci to open a PowerShell prompt inside the container
Remove-BcContainer -containerName hostedagent-bluebearwaste-ci to remove the container again
docker logs hostedagent-bluebearwaste-ci to retrieve information about URL's again

Creating container took 1578 seconds

```

**Screenshots**
![image](https://github.com/microsoft/navcontainerhelper/assets/16802407/31490458-5559-4016-b388-6b096d7d7099)

--- a/ContainerHandling/New-NavContainer.ps1
+++ b/ContainerHandling/New-NavContainer.ps1
@@ -1657,8 +1657,8 @@ if (!(Test-Path "c:\navpfiles\*")) {
     if ($assignPremiumPlan -and !$restoreBakFolder -and !$skipDatabase) {
         if (!(Test-Path -Path "$myfolder\SetupNavUsers.ps1")) {
             ('# Invoke default behavior
-              . (Join-Path $runPath $MyInvocation.MyCommand.Name)
-            ') | Set-Content -Path "$myfolder\SetupNavUsers.ps1"
+. "c:\run\SetupNavUsers.ps1"
+') | Set-Content -Path "$myfolder\SetupNavUsers.ps1"
         }
      
         if ($version.Major -ge 15) {
@@ -1698,13 +1698,13 @@ Get-NavServerUser -serverInstance $ServerInstance -tenant default |? LicenseType
 
     if (!(Test-Path -Path "$myfolder\SetupVariables.ps1")) {
         ('# Invoke default behavior
-          . (Join-Path $runPath $MyInvocation.MyCommand.Name)
-        ') | Set-Content -Path "$myfolder\SetupVariables.ps1"
+. "c:\run\SetupVariables.ps1"
+') | Set-Content -Path "$myfolder\SetupVariables.ps1"
     }
     if (!(Test-Path -Path "$myfolder\HelperFunctions.ps1")) {
         ('# Invoke default behavior
-          . (Join-Path $runPath $MyInvocation.MyCommand.Name)
-        ') | Set-Content -Path "$myfolder\HelperFunctions.ps1"
+. "c:\run\HelperFunctions.ps1"
+') | Set-Content -Path "$myfolder\HelperFunctions.ps1"
     }
 
     if ($version.Major -ge 24 -and $genericTag -eq [System.Version]"1.0.2.15") {
@@ -1714,12 +1714,6 @@ Get-NavServerUser -serverInstance $ServerInstance -tenant default |? LicenseType
     if ($version.Major -ge 24 -and $genericTag -lt [System.Version]"1.0.2.17") {
         Download-File -source "https://raw.githubusercontent.com/microsoft/nav-docker/4123cbcd197df57a5d94fa15b976356c11f4bcac/generic/Run/240/navinstall.ps1" -destinationFile (Join-Path $myFolder "navinstall.ps1")
         Download-File -source "https://raw.githubusercontent.com/microsoft/nav-docker/4123cbcd197df57a5d94fa15b976356c11f4bcac/generic/Run/navstart.ps1" -destinationFile (Join-Path $myFolder "navstart.ps1")
-    }
-
-    if ($version.Major -ge 24) {
-        ('
-if (!(Get-Command "invoke-sqlcmd" -ErrorAction SilentlyContinue)) { try { Write-Host "Installing SqlServer module"; Install-package SqlServer -Force -RequiredVersion 21.1.18256 | Out-Null } catch { Write-Host "Failed to install SqlServer module" } }
-') | Add-Content -Path "$myfolder\HelperFunctions.ps1"
     }
 
     if ($version.Major -ge 15 -and $version.Major -le 18 -and $genericTag -ge [System.Version]"1.0.2.15") {

--- a/ReleaseNotes.txt
+++ b/ReleaseNotes.txt
@@ -2,6 +2,7 @@
 Issue 3452 Error when running Run-AlValidation : Unable to load one or more of the requested types. Retrieve the LoaderExceptions property for more information.
 Issue 3454 DockerDo : Cannot bind argument to parameter 'imageName' because it is an empty string.
 Issue 3354 Regression : Compile Folder removal of symbols folder fails: The process cannot access the file
+Issue 3460 Do not install SqlServer PowerShell module
 
 6.0.12
 Issue 3432 New-BcContainerWizard stopped working after generic images with ltsc2016, ltsc2019 and ltsc2022 was introduced

